### DISCUSSION TITLE
Script as SYSTEM user

### DISCUSSION THREAD

**Discussion by glSPICE** [Windos/BurntToast#154] [Q&A]
Hi! I use Windows 10 task scheduler to start Powershell as SYSTEM to hide console window and Java application that I execute inside script (using Start-Process). How can I use BurntToastNotification in this situation? I do not get notifications, because the user is SYSTEM, when I start as current USER everything is fine but every 10 minutes I see console and Java ((

**Comment by Windos** [comment]
To kick the toast over to the user context from SYSTEM, you can use @KelvinTegelaar's [RunAsUser](https://www.powershellgallery.com/packages/RunAsUser) ([GitHub](https://github.com/KelvinTegelaar/RunAsUser/)). This was written specifically for use cases like this, though normally from an RMM system that tends to run its code under the SYSTEM context.

You can get some more context on this from this blog post: [Monitoring with PowerShell: Notifying users of Windows Updates](https://www.cyberdrain.com/monitoring-with-powershell-notifying-users-of-windows-updates/).

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
